src/index.js

```diff
@@ -59,12 +60,11 @@ class RobotSession {
     this.logger.info(`Fetching config for robot ${this.robotId} for apiKey ${this.apiKey.substr(0, 3)}...`);
 
     const params = {
-      apiKey: this.apiKey,
+      appKey: this.apiKey,
```

Comment:
Did you forgot to refactor `apiKey`?

`appKey: this.appKey`

Reply:
No, the thing is that externally we have decided to call this "API key", but internally the server still uses `appKey` :/